hal/src/main/native/athena/REVPH.cpp

```diff
@@ -230,17 +230,19 @@ HAL_REVPHHandle HAL_InitializeREVPH(int32_t module,
 
   // Start closed-loop compressor control by starting solenoid state updates
   HAL_SendREVPHSolenoidsState(hph.get(), status);
-
+  if (*status != 0) {
```

Comment:
This still results in the exact same problem. Only the status variable is checked to throw the exception. The CTRE code could also throw an exception here. So we'd need to fix both.

Reply:
What does the CTRE code have to do with this code?

Reply:
The issue that was being reported was that a reboot after the bus disconnects results in a code reboot loop, because it keeps crashing. The same issue exists in both this and CTRE.